WalletWasabi.Tests/Helpers/StringNoWhiteSpaceEqualityComparer.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+
+namespace WalletWasabi.Tests.Helpers
+{
+	public class StringNoWhiteSpaceEqualityComparer : IEqualityComparer<string>
+	{
+		public bool Equals([AllowNull] string x, [AllowNull] string y)
```

Comment:
Does this really work when `x` or `y` is null? How does it work?

Reply:
Fixed